.codeclimate.yml

```diff
@@ -34,6 +34,12 @@ plugins:
   eslint:
     enabled: true
     channel: "eslint-4"
+  duplication:
```

Comment:
Does it help? As I said, I can go and ignore warnings for existing code and any new code which gets triggered should be reviewed (and can be ignored if it's a false positive). My concern is this would mask the initial trigger.

Reply:
Yes. Helped in this case. Default threshold is 45.
https://docs.codeclimate.com/docs/default-analysis-configuration

Reply:
In that case, this is fine!